libr/core/cbin.c

```diff
@@ -3263,14 +3263,14 @@ static void bin_pe_resources(RCore *r, int mode) {
 		} else if (IS_MODE_RAD (mode)) {
 			r_cons_printf ("f resource.%d %d 0x%08"PFMT32x"\n", index, size, vaddr);
 		} else if (IS_MODE_JSON (mode)) {
-			r_cons_printf ("%s{\"name\":%d,\"index\":%d, \"type\":\"%s\","
+			r_cons_printf ("%s{\"name\":\"%s\", \"index\":%d, \"type\":\"%s\", "
```

Comment:
will be good to switch to use the pj.c api in here too.maybe in another pr? or do you want to finish it in here?

Reply:
Seem straightforward to change. I'll do it in this PR.